respy/tests/codes/auxiliary.py

```diff
@@ -130,23 +127,18 @@ def compare_est_log(base_est_log):
 def get_floats(line):
     """ This extracts the floats from the line
     """
-    list_ = shlex.split(line)[1:]
-    rslt = []
+    list_ = shlex.split(line)
+    result = []
     for val in list_:
-        if val == "---":
-            val = HUGE_FLOAT
-        else:
-            val = float(val)
-        rslt += [val]
-    return rslt
+        if not isinstance(val, str):
+            result.append(float(val))
+    return result
```

Comment:
Maybe ``[float(val) for val in list_ if not isinstance(val, str)]``?

Reply:
Yes, changed it. Thanks!